nntrainer/src/layer.cpp

```diff
@@ -198,6 +198,15 @@ void Layer::setProperty(const PropertyType type, const std::string &value) {
       bias_initializer = (WeightInitializer)parseType(value, TOKEN_WEIGHT_INIT);
     }
     break;
+  case PropertyType::input_layers:
+    if (!value.empty()) {
+      std::regex reg("\\,+");
+      std::vector<std::string> concat_layers = split(value, reg);
+      num_inputs = 1;
+      if (concat_layers.size() > 1)
```

Comment:
what does check means? If the parse string reveals size is 0, then we should set num_inputs to that and init would give it out as a error. But here, it seems that in case concat_layers.size() is 0, then num_inputs=1 but then what does it point to?

Reply:
yes. you are right. But there is layer which not need input like input_layers. It is different story if we are using databuffer as an layer, then input_layer need input_layers = 'databuffer_name'. That means every layer needs layer_inputs. That's what I plan to implement. set num_input=1 is just temporary and once every thing is implemented properly, I will fix it.

Reply:
Then lets add a `TODO/FIXME` here.